editor/js/components/arcontroller.component.js

```diff
@@ -348,6 +348,9 @@ ArControllerComponent.prototype.stopAR = function(){
     if(arBackground)
         LS.GlobalScene.root.removeChild(arBackground);
     this.initVideo = true;
+
+    if(this.arController)
+        this.arController.dispose;
```

Comment:
Does this is correct? but i don't receive any error now...

Reply:
i think that is not required because is disposed some lines above, you can ceck easily adding some console.log(this.arController); before dispose and after. I will remove this.